test/corecel/math/ConvexHullFinder.test.cc

```diff
@@ -0,0 +1,237 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file corecel/math/ConvexHullFinder.test.cc
+//---------------------------------------------------------------------------//
+#include "corecel/math/ConvexHullFinder.hh"
+
+#include "corecel/cont/Range.hh"
+
+#include "celeritas_test.hh"
+
+namespace celeritas
+{
+namespace test
+{
+//---------------------------------------------------------------------------//
+/*
+ * Test harness for ConvexHullFinder
+ */
+class ConvexHullFinderTest : public ::celeritas::test::Test
+{
+  public:
+    using CHF = ConvexHullFinder<double>;
+    using Points = CHF::Points;
+    using ConcaveRegions = CHF::ConcaveRegions;
+
+    void compare_convex_hulls(Points expected, Points actual)
```

Comment:
```suggestion
    void compare_convex_hulls(Points const& expected, Points const& actual)
```
and also const references below.

It might be easier for both cases to just provide a `linearize` function that turns the reference and actual into vectors that you can then EXPECT_VEC_EQ.

Also rename this function (and the next) `expect_eq` since that's a better description of what they're doing.

Reply:
> It might be easier for both cases to just provide a linearize function that turns the reference and actual into vectors that you can then EXPECT_VEC_EQ.

While this would be fine for the VecReal2, it could cause problems for VecVecReal2 as:

```
{{point_a, point_b, point_c}, {point_e, point_f, point_g, point_h}}
```

and

```
{{point_a, point_b, point_c, point_e}, {point_f, point_g, point_h}}
```

would return equal. It's not impossible that a bug introduced to the Graham Scan could result in something like this.

Reply:
Fixed name and changed to const ref (the latter was of course an oversight)